internal/ostree/ostree.go

```diff
@@ -80,22 +80,15 @@ func ResolveRef(location, ref string) (string, error) {
 // without URL results in a ParameterComboError. Failure to resolve the
 // checksum results in a ResolveRefError.
 //
-// If Ref is not specified in the RequestParams, the defaultRef is used.
-// If Parent is not specified in the RequestParams, the value of Ref is used
-// (which will be defaultRef if neither is specified).
+// If Parent is not specified in the RequestParams, the value of Ref is used.
 //
 // If any ref (Ref or Parent) is malformed, the function returns with a RefError.
-func ResolveParams(params RequestParams, defaultRef string) (ref, checksum string, err error) {
-	// Determine value of ref
+func ResolveParams(params RequestParams) (ref, checksum string, err error) {
```

Comment:
Reading this now I notice that returning the `ref` is essentially useless (like when we used to return the URL).  Maybe we can drop `ref` from the returns and only return `checksum` and `error`, and since the caller is responsible for using the default ref if one isn't specified by the user, then it's also responsible for using it directly instead of getting it straight back from this function.

Reply:
We could also rename the function maybe?  `ResolveRef()` is the other function in the package that does the actual resolving.  This function is now essentially a wrapper around that which also runs refs through `VerifyRef()`.  We could call it `VerifyAndResolve()` but I hate that.

Reply:
Never mind, don't rename the function.  Also we can drop the `ref` from the return in a follow-up.  It's nbd.